vochain/vochaininfo/vochaininfo.go

```diff
@@ -68,14 +72,17 @@ func (vi *VochainInfo) updateCounters() {
 	mempoolSize.Set(uint64(vi.vnode.MempoolSize()))
 }
 
-// Height returns the current number of blocks of the blockchain
+// Height returns the current number of blocks of the blockchain.
```

Comment:
???? :rofl: if you really want to lint this, make it on a separate commit linting the thousand comments with missing dot